pages/linux/dmidecode.md

```diff
@@ -0,0 +1,23 @@
+# dmidecode
+
+> Display the DMI (some say SMBIOS) table contents in a human-readable format as root user.
+
+- Show all DMI table contents:
+
+`sudo dmidecode`
+
+- Show the BIOS version:
+
+`sudo dmidecode -s bios-version`
+
+- Show the system serial number:
+
+`sudo dmidecode system-serial-number`
```

Comment:
This outputs the contents of 96 tables for me on my _Dell Inspiron 15_. Maybe it's system specific.

Reply:
Oops. an error. Should be `sudo dmidecode -s system-serial-number`. I'll fix it.